app/assets/stylesheets/playground.css

```diff
@@ -1,5 +1,5 @@
 /*
- *= require swagger-lib
+ *= require swagger-ui-dist
```

Comment:
If I'm understanding correctly, by using `swagger-ui-dist` rather than just `swagger-ui-lib` you're pulling a pre-built set of swagger files from NPM, rather than pulling it in as a managed dependency, right? 

I've not yet done any kind of front-end work within a Ruby on Rails context, is this standard? Do we generally want to avoid NPM? Is there some alternative approach possible where we add `swagger-ui-lib` to a `package.json` file and let NPM take care of it, rather than just dumping a prebuilt version into the assets folder?

Reply:
The old way was cherry picking files from source. Since then swagger-ui released a self-contained distribution without dependencies, and is now the recommended way of using that in Rails. Npm is just a download mechanism here, I just couldn't find a tarball or something like that to get. 

Generally we don't add node package managers into Rails apps :)